src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return (
+        `name: ${this.name},`
+        + ` chip version: ${this.version},`
+        + ` wheels: ${this.wheels}`
+      );
+    },
+    get location() {
+      return (
+        `${this.name}:`
+        + ` x=${this.coords.x},`
+        + ` y=${this.coords.y}`
+      );
+    },
+    goForward(value = 1) {
```

Comment:
```suggestion
    },
    
    goForward(value = 1) {
```

better for readability to add empty line between